velox/expression/FunctionSignature.cpp

```diff
@@ -251,6 +251,31 @@ FunctionSignature::FunctionSignature(
   validate(variables_, returnType_, argumentTypes_, constantArguments_);
 }
 
+std::string AggregateFunctionSignature::toString() const {
+  std::vector<std::string> arguments;
+  auto& argumentTypes = this->argumentTypes();
+  auto& constantArguments = this->constantArguments();
+  auto size = argumentTypes.size();
+  arguments.reserve(size);
+
+  for (auto i = 0; i < size; ++i) {
```

Comment:
This looks like a copy-paste from somewhere. Would it be possible to refactor to avoid duplicating this code?